src/overlays/actors/ovl_Bg_Mori_Bigst/z_bg_mori_bigst.c

```diff
@@ -212,17 +210,15 @@ void BgMoriBigst_SetupStalfosPairFight(BgMoriBigst* this, PlayState* play) {
         this->dyna.actor.child = NULL;
         this->dyna.actor.home.rot.z++;
     } else {
-        // "Warning: 3-1 Stalfos failure"
-        PRINTF("Warning : 第３-1スタルフォス発生失敗\n");
+        PRINTF(T("Warning : 第３-1スタルフォス発生失敗\n", "Warning : 3-1 Stalfos generation failure\n"));
     }
     stalfos2 = Actor_SpawnAsChild(&play->actorCtx, &this->dyna.actor, play, ACTOR_EN_TEST, 170.0f, 827.0f, -3260.0f, 0,
                                   0, 0, 5);
     if (stalfos2 != NULL) {
         this->dyna.actor.child = NULL;
         this->dyna.actor.home.rot.z++;
     } else {
-        // "Warning: 3-2 Stalfos failure"
-        PRINTF("Warning : 第３-2スタルフォス発生失敗\n");
+        PRINTF(T("Warning : 第３-2スタルフォス発生失敗\n", "Warning : 3-2 Stalfos generation failure\n"));
```

Comment:
This is a general note, but "generate" is used by google translate where "spawn" probably fits better. I see some cases in other strings where spawn does infact get used even though its the same japanese word. So either now or later I think we should just use spawn in these cases.

Reply:
+1